website/blog/2023-05-01-evolving-data-engineer-craft.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Data engineers + dbt v1.5: Evolving the craft for scale"
+description: "Where have we been? Where are we going? What does the future hold for data engineers? How can dbt help you get there?"
+slug: evolving-data-engineer-craft
+
+authors: [sung_chung, kira_furuichi]
+
+tags: [data ecosystem]
```

Comment:
think we should change the tag to either `dbt tutorials` or `analytics craft`, `dbt product updates`!

Reply:
Let's go with dbt product updates!